R/read_vpts.R

```diff
@@ -1,241 +1,144 @@
-#' Read a time series of vertical profiles (`vpts`) from file
-#'
-#' @param file A text file containing the standard output (stdout) generated
-#' by vol2bird (or the package function `calculate_vp`).
-#' @param lon numeric. Longitude of the radar in decimal degrees.
-#' @param lat numeric. Latitude of the radar in decimal degrees.
-#' @param height numeric. Height above sea level of the radar antenna in meters.
-#' @param radar A string containing a radar identifier.
-#' @param wavelength Radar wavelength in cm, or one of 'C' or 'S' for C-band
-#' and S-band radar, respectively, in which case C-band wavelength is assumed
-#' to be 5.3 cm and S-band wavelength 10.6 cm
-#' @param sep the field separator character, see [utils::read.table]
-#'
-#' @return An object inheriting from class `vpts`, see
-#' [`vpts()`][summary.vpts] for details.
+#' Read time series of vertical profiles (`vpts`) from file(s)
 #'
+#' Reads `vpts` data from one or more files.
+#' The following file formats are supported (but cannot be mixed):
+#' - [VPTS CSV](https://aloftdata.eu/vpts-csv/).
+#' - [ODIM bird profile](https://github.com/adokter/vol2bird/wiki/ODIM-bird-profile-format-specification).
+#' @param files Path(s) to one or more files containing vpts data.
+#' @param ... Additional arguments for backward compatibility, passed to `read_stdout`.
+#' @return `vpts` object.
```

Comment:
currently only reads of stdout and h5 formats returns a true `vpts`object, the vpts csv function returns a `data.frame` that is not a true bioRad `vpts` object. So we need to document in which cases the return value is a `vpts` and in which cases it is a `data.frame`

Reply:
The goal (for now) is to always return a true bioRad `vpts` object.

Reply:
`read_vpts()` should now always return a true `vpts` object

Reply:
`read_vpts()` should now always return a true `vpts` object even from csv